torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -463,14 +463,21 @@ void ONNXSetDynamicInputShape(
         continue;
       }
 
-      auto shape = input_tensor_type->symbolic_sizes().sizes().value();
+      auto shape_ref = input_tensor_type->symbolic_sizes().sizes();
+      TORCH_CHECK(
+          shape_ref.has_value(), "Input tensor shape should have value.");
+      auto shape = shape_ref.value();
```

Comment:
Do we have tests to verify this part?

Reply:
No, currently we don't have a proper designed test for it. (1) All existing tests passed (2) Will add the test once we find a proper one.